docker-gen.go

```diff
@@ -58,6 +58,13 @@ type Volume struct {
 	ReadWrite bool
 }
 
+type Mount struct {
+	Source      string
+	Destination string
+	Mode        string
+	RW          bool
```

Comment:
On Sat, Sep 12, 2015 at 02:45:21PM -0700, Jason Wilder wrote:
>    In docker-gen.go:
>
>  > @@ -58,6 +58,13 @@ type Volume struct {
>  >    ReadWrite bool
>  >  }
>  >
>  > +type Mount struct {
>  > +  Source      string
>  > +  Destination string
>  > +  Mode        string
>  > +  RW          bool
>
>    docker-gen exposes it's own API that I'd like to keep consistent. It's not
>    the same as go-dockerclient even though it uses it.
>

If that is the case, since Volumes and Mounts are mutually exclusive, how
about not adding a new `Mounts` structure at all and filling in the `Volumes`
structure w/ the mounts info:

	for _, v := range container.Mounts {
		runtimeContainer.Volumes[v.Destination] = Volume{
			HostPath:  v.Source,
			Path:      v.Destination,
			ReadWrite: v.RW,
		}
	}

That way the docker-gen api doesn't change at all, regardless of the
go-dockerclient version. Also, it means the only change to the source is the
addition of the above. No changes to data structures, etc.

rick